src/resources.py

```diff
@@ -17,6 +18,12 @@
 from helpers import request, write_data_to_file, remove_file, timestamp, unique_filename, CONTENT_TYPE_TEXT, \
     CONTENT_TYPE_BASE64_BINARY, execute, WATCH_SERVER_TIMEOUT, WATCH_CLIENT_TIMEOUT
 
+log_level = os.getenv("LOG_LEVEL", logging.INFO)
```

Comment:
I don't like that the environment variable is read multiple times. Can't we centralize this and provide a parsed/validated log level in a central place which gets used by all instantiated loggers then?

Reply:
We might centralize in `helpers.py` considering `resources.py`and `sidecar.py` imports `helpers.py`. 

Or we can create a dedicated `logger.py` to initialize the logger and define error handling as suggested by @schans.

Let me know what you prefer !

Reply:
I'd go for `helpers.py` :)

Reply:
Done !